.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*] # rules that follow apply to files with any extension
+insert_final_newline = false
```

Comment:
I think we're inserting a final new line everywhere, or in most places, but I could be wrong.

(I'm out and about at the moment so I can't check.)

Reply:
If that's the case then this must be set to true.

Reply:
Yep, I just checked.  I changed that setting.

Reply:
👍